src/deposit.cairo

```diff
@@ -91,8 +92,7 @@ mod Deposit {
             * borrow_factor.into()
             / ZK_SCALE_DECIMALS)
             - total_debt.into();
-        let withdraw_amount = free_amount * debt_token_price.into() / debt_decimals.into();
-        withdraw_amount
+        free_amount * debt_token_price.into() / debt_decimals.into() * repay_const.into() / 100
```

Comment:
can you add parentheses () around the expression so the priority of operations is clear?